packages/ember/tests/component_registration_test.js

```diff
@@ -39,9 +40,13 @@ function cleanup() {
 }
 
 function cleanupHelpers() {
+  var included;
+
   keys(helpers).
     forEach((name) => {
-      if (!originalHelpers.contains(name)) {
+      included = isEnabled('ember-runtime-enumerable-includes') ? originalHelpers.includes(name) : originalHelpers.contains(name);
```

Comment:
Unfortunately the ternary syntax is not supported by feature flag stripping. Please rewrite this to use an `if` statement.